python/ambassador/ir/irambassador.py

```diff
@@ -410,7 +410,7 @@ def add_mappings(self, ir: 'IR', aconf: Config):
                 name = "internal_%s_probe_mapping" % name
 
                 mapping = IRHTTPMapping(ir, aconf, rkey=self.rkey, name=name, location=self.location,
-                                        timeout_ms=10000, **cur)
+                                        timeout_ms=10000, host="*", **cur)
```

Comment:
Same deal. Some bewildering errors if we don't do this -- also note that a user who doesn't want `host: *` for these `Mapping`s for the diagnostics, `check_ready`, etc., can set those things as disabled and add `Mapping`s by hand.